packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-legend-label/gux-form-field-legend-label.tsx

```diff
@@ -11,14 +11,19 @@ export const GuxFormFieldLegendLabel: FunctionalComponent<
   GuxFormFieldLegendLabelProps
 > = ({ position, required }, children): VNode => {
   return (
-    <legend
+    <div
```

Comment:
legend does not work with Flexbox in Safari. This is the workaround. I have submitted feedback to Apple about the bug

Reply:
I have concerns about this because the `legend` element will no longer be the first direct child of the `fieldset` element which I think it has to be.

Would reversing it still work as a workaround?
```
<legend>
  <span class="flex-stuff">
    {children}
  <span/>
</legend>
```

Reply:
Nope. I tried. The actual flexbox is the fieldset wrapper around the legend and input. Legend just ignores it entirely and shows up above the text input instead of beside

Reply:
Ya maybe look into something like that so :(

Reply:
Managed to get it to work